.github/workflows/deploy_production.yml

```
@@ -0,0 +1,93 @@
+name: Deploy ALICE Production
+
+on:
+    # Run this workflow on push to production-release tag (via chatops)
+    push:
+        tags:
+          - production-release
+
+    # Allow running this workflow manually from the Actions tab
+    workflow_dispatch:
+
+jobs:
+  deploy_staging_branch:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: azure/login@v1
+      with:
+          creds: ${{ secrets.AZURE_STATIC_SITES }}
+
+    - name: Node.js build
+      uses: actions/setup-node@v1
+      with:
+        node-version: '14.x'
+    - run: yarn install
+    - run: yarn build
+      env:
+        REACT_APP_ENV: development
```

I think this needs to be production.
```suggestion
        REACT_APP_ENV: production
```

---

I thought it had to be development so that the dev deps would be included for the build step?

---

This env var is just being used to set which panoptes API endpoint to use since react-scripts is controlling NODE_ENV for us and NODE_ENV is what npm will use to determine what packages to install.

---

ah-hah